app/views/business_contact_forms/_form.slim

```diff
@@ -0,0 +1,29 @@
+.container.contact
+  h1.p-3
+    = t('.title')
+  .row.justify-content-center
+    .col-md-8.col-11
+      = simple_form_for @business_contact_form do |f|
+        .row.gx-2
+          .col-lg-6.col-12
+            = f.input :first_name, placeholder: t('.first_name')
+          .col-lg-6.col-12
+            = f.input :last_name, placeholder: t('.last_name')
+        = f.input :company_name, placeholder: t('.company_name')
+        .row.gx-2
+          .col-lg-6.col-12
+            = f.input :email, placeholder: t('.email'), required: true
+          .col-lg-6.col-12
+            = f.input :phone_number, placeholder: t('.phone_number')
+        = f.input :industry, placeholder: t('.industry')
+        h2.pt-3
+          = t('.area_of_interest')
+        = f.input :area_of_interest, as: :check_boxes, collection: t('business_contact_forms.areas_of_interest'), label_method: :second, value_method: :first, legend_tag: false
+        = f.input :message, placeholder: t('.message'), as: :text, input_html: { class: 'mt-3', style: 'height: 240px' }
+        = f.input :file, label: false, as: :file, hint: t('.maximum_size')
+        h2.pt-3
+          = t('.nda')
+        = f.input :nda, label: t('.true'), as: :boolean
```

Comment:
Proposed part of form (the content is of secondary importance):
```
[ ] Do you need an NDA?
```
Checkbox name: `need_nda`